src/modules/localization/domain/kalman_filter/Dynamics.cpp

```diff
@@ -0,0 +1,40 @@
+#include "Dynamics.h"
+
+Dynamics::Dynamics(){}
```

Comment:
Since it is a default constructor, no need to declare and define(Check locally and update)